src/azure-cli/azure/cli/command_modules/appconfig/_kv_helpers.py

```diff
@@ -235,31 +235,74 @@ def __write_kv_and_features_to_file(file_path, key_values=None, features=None, f
 
 # Config Store <-> List of KeyValue object
 
-
-def __read_kv_from_config_store(cmd, name=None, connection_string=None, key=None, label=None, prefix_to_remove="", prefix_to_add=""):
-    from .keyvalue import list_key
+def __read_kv_from_config_store(cmd,
+                                name=None,
+                                connection_string=None,
+                                key=None,
+                                label=None,
+                                datetime=None,
+                                fields=None,
+                                top=None,
+                                all_=False,
```

Comment:
>all_=False, [](start = 32, length = 11)

all_ should be True by default because we are not passing this explicitly in import/export calls.

Reply:
Good catch. Will fix. 

---
In reply to: [405720099](https://github.com/Azure/azure-cli/pull/12893#discussion_r405720099) [](ancestors = 405720099)